mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -985,7 +989,7 @@ trait Completions { this: MetalsGlobal =>
           allParams.exists(param => param.name.startsWith(prefix))
         val isExplicitlyCalled = suffix.startsWith(prefix)
         val hasParamsToFill = allParams.count(!_.hasDefault) > 1
-        if ((shouldShow || isExplicitlyCalled) && hasParamsToFill) {
+        if ((shouldShow || isExplicitlyCalled) && hasParamsToFill && clientSupportsSnippets) {
```

Comment:
this disables the `Autofill with default values` completion when the client does not support snippets, as per @tgodzik. Please review that it makes sense and that we can't find a good completion that works without snippets

Reply:
Let's disable it for now, I will think it over to see if it makes sense without the snippets.